<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>801</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Search</string>
    </attribute>
    <widget class="QLineEdit" name="search_input_name">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>771</width>
       <height>24</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Search Name (regex)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="search_button_goto">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>480</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Go to</string>
     </property>
    </widget>
    <widget class="QTreeWidget" name="search_res_tree">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>771</width>
       <height>381</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SelectionMode::ExtendedSelection</enum>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
    <widget class="QCheckBox" name="search_is_specific_url">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Specific URL:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="search_input_url">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>671</width>
       <height>24</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Search URL (regex)</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>Replace</string>
    </attribute>
    <widget class="QLineEdit" name="replace_input_search">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>24</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="replace_input_replace">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>10</y>
       <width>351</width>
       <height>24</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Replace</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="replace_input_url">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>551</width>
       <height>24</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>regex</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="replace_match_url">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>221</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>If the URL matches the following rule:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>375</x>
       <y>8</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>--&gt;</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>351</width>
       <height>391</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>349</width>
        <height>389</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>70</y>
       <width>351</width>
       <height>391</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>349</width>
        <height>389</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="replace_button_commit">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>480</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Commit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="replace_button_reset">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>480</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="replace_is_live_preview">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>480</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Live preview</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>View</string>
    </attribute>
    <widget class="QTreeWidget" name="view_tree">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>771</width>
       <height>501</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
